packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -0,0 +1,169 @@
+import { 
+  Component, 
+  Element, 
+  Host, 
+  h, 
+  Prop, 
+  Build, 
+  forceUpdate,
+  Fragment,
+  Event,
+  EventEmitter
+} from '@stencil/core';
+import i18next from 'i18next';
+
+if (Build.isTesting) {
+  // Make i18next.t() return the key instead of the value
+  i18next.init({ lng: 'cimode' });
+}
+
+@Component({
+  tag: 'va-file-input',
+  styleUrl: 'va-file-input.css',
+  shadow: true,
+})
+
+export class VaFileInput {
+  @Element() el: HTMLElement;
+
+  /**
+   * The label for the button input.
+   */
+  @Prop() label?: string;
+
+  /**
+   * The name for the input element.
+   */
+  @Prop() name?: string;
+
+  /**
+   * The text displayed on the button.
+   */
+  @Prop() buttontext?: string;
```

Comment:
Should we give this a default value of "Upload Document"?

Reply:
This is minor, but how about if we update this to use camel case or rename to `text`?

Reply:
If we don't supply a default value, we should make it a required prop and reflect that in our Storybook.

Reply:
@andresriveratoro There is some discussion about if we should provide this as an attribute at all and rather set a static text value. If the decision has been made to allow this text to be dynamic, I'll definitely come back around and address your suggestions.